packages/circuit-ui/components/CardList/CardList.js

```diff
@@ -24,6 +24,8 @@ const Wrapper = styled(Card)`
   label: cardlist__wrapper;
 
   padding: 0;
+  border-radius: ${(p) => p.theme.borderRadius.mega};
```

Comment:
👀 new pattern without destructuring the theme?

Fine by me, it avoids some curly brackets and is slightly shorter. I can start using it as well 🙂

Reply:
I occasionally use this pattern when the theme is only needed for 1 or 2 properties. I find it easier to read.